single_tech_samples/datafactory/sample1_cicd/README.md

```diff
@@ -0,0 +1,282 @@
+
+#### Software pre-requisites
+
+- For Windows users, [Windows Subsystem For Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+- [az cli 2.6+](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- [az cli - application insights extension](https://docs.microsoft.com/en-us/cli/azure/ext/application-insights/monitor/app-insights?view=azure-cli-latest)
+  - To install, run `az extension add --name application-insights`
+- [Azure DevOps CLI](https://marketplace.visualstudio.com/items?itemName=ms-vsts.cli)
+  - To install, run `az extension add --name azure-devops`
+- [jq](https://stedolan.github.io/jq/)
+
+### Setup and Deployment
+
+> **IMPORTANT NOTE:** As with all Azure Deployments, this will **incur associated costs**. Remember to teardown all related resources after use to avoid unnecessary costs. See [here](#deployed-resources) for list of deployed resources.
+
+This deployment was tested using WSL 2 (Ubuntu 20.04)
+
+1. **Initial Setup**
+   1. Ensure that:
+      - You are logged in to the Azure CLI. To login, run `az login`.
+      - Azure CLI is targeting the Azure Subscription you want to deploy the resources to.
+         - To set target Azure Subscription, run `az account set -s <AZURE_SUBSCRIPTION_ID>`
+      - Azure CLI is targeting the Azure DevOps organization and project you want to deploy the pipelines to.
+         - To set target Azure DevOps project, run `az devops configure --defaults organization=https://dev.azure.com/<MY_ORG>/ project=<MY_PROJECT>`
+   2. **Fork** this repository into a new Github repo.
+   3. Set the following **required** environment variables:
+       - **GITHUB_REPO** - Name of your imported github repo in this form `<my_github_handle>/<repo>`
+       - **GITHUB_PAT_TOKEN** - a Github PAT token. Generate them [here](https://github.com/settings/tokens). This requires "repo" scope.
+
+       Optionally, set the following environment variables:
+       - **RESOURCE_GROUP_LOCATION** - Azure location to deploy resources. *Default*: `westus`.
+       - **AZURE_SUBSCRIPTION_ID** - Azure subscription id to use to deploy resources. *Default*: default azure subscription. To see your default, run `az account list`.
+       - **RESOURCE_GROUP_NAME_PREFIX** - name of the resource group. This will automatically be appended with the environment name. For example: `RESOURCE_GROUP_NAME_PREFIX-dev-rg`. *Default*: mdwdo-ado-${DEPLOYMENT_ID}.
+       - **DEPLOYMENT_ID** - string appended to all resource names. This is to ensure uniqueness of azure resource names. *Default*: random five character string.
+       - **AZDO_PIPELINES_BRANCH_NAME** - git branch where Azure DevOps pipelines definitions are retrieved from. *Default*: main.
+
+      To further customize the solution, set parameters in `arm.parameters` files located in the `infrastructure` folder.
+
+2. **Deploy Azure resources**
+   1. `cd` into the `single_tech_samples/sample1_cicd/datafactory` folder of the repo
```

Comment:
This should be "single_tech_samples/datafactory/sample1_cicd"

Reply:
Good catch! Wrong cursor in copy paste process 🙃

Reply:
Fixed